packages/bind/spec/bindingDependencyBehaviors.ts

```diff
@@ -375,26 +376,27 @@ describe('Binding dependencies', function () {
       applyBindings(vm, testNode);
       expect(vm.getSubscriptionsCount()).toEqual(1);
 
-      expect(testNode.childNodes[0].childNodes[0].value).toEqual('My prop value');
+      const child = testNode.childNodes[0].childNodes[0] as HTMLInputElement;
+      expect(child.value).toEqual('My prop value');
 
-            // a change to the input value should be written to the model
-      testNode.childNodes[0].childNodes[0].value = 'some user-entered value';
-      triggerEvent(testNode.childNodes[0].childNodes[0], 'change');
+      // a change to the input value should be written to the model
+      child.value = 'some user-entered value';
+      triggerEvent(child, 'change');
       expect(vm().someProp).toEqual('some user-entered value');
             // a click should use correct view model
-      triggerEvent(testNode.childNodes[0].childNodes[1], 'click');
+      triggerEvent(testNode.childNodes[0].childNodes[1] as Element, 'click');
       expect(clickedVM).toEqual(vm());
 
             // set the view-model to a new object
       vm({ someProp: observableConstructor('My new prop value'), checkVM: checkVM });
-      expect(testNode.childNodes[0].childNodes[0].value).toEqual('My new prop value');
+      expect(child.value).toEqual('My new prop value');
```

Comment:
To make sure it has the latest DOM structure, child should be a function i.e. 

```ts
const child = () => testNode.childNodes[0].childNodes[0] as HTMLInputElement
```

... because part of the assertion here is the DOM structure i.e. `testNode.childNodes[0].childNodes[0]` may no longer be `child` here, and if that were the case we'd want this test to fail.  That failure wouldn't be visible if `input` is a reference to the input.

Reply:
Done